integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/BearerTokenAuthorizationTest.java

```diff
@@ -61,6 +62,7 @@ public void testSecureAccessSuccessPreferredUsername() {
     }
 
     @Test
+    @Disabled
```

Comment:
We probably set the value of the annotation to specify the reason why they are disabled, but let's not waste CI :)

Reply:
@geoand 

> We probably set the value of the annotation to specify the reason why they are disabled, but let's not waste CI :)

Sorry, you are right, I just did a quick disabling yesterday evening to minimize the disruption, and we'll be discussing now how to re-enable them effectively asap

Reply:
No problem